skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -104,18 +104,52 @@ def test_lr_callback_batch_steps_correctly(
             policy,
             kwargs,
     ):
-        num_examples = 1000
         batch_size = 100
-        max_epochs = 2
+        max_epochs = 1
 
         X, y = classifier_data
+        num_examples = len(X)
+
         lr_policy = LRScheduler(policy, **kwargs)
         net = NeuralNetClassifier(classifier_module(), max_epochs=max_epochs,
                                   batch_size=batch_size, callbacks=[lr_policy])
         net.fit(X, y)
-        expected = (num_examples // batch_size) * max_epochs - 1
+        net.partial_fit(X, y)
+        expected = int(1.6 * num_examples / batch_size) * max_epochs
```

Comment:
Can you clarify this expression? I find it non-obvious (where does the 1.6 come from?) and having `max_epochs` here with `max_epochs=1` doesn't make sense.

Reply:
The PR was updated to break the steps up when calculating the expected batch_idx.